### PR TITLE
Update demo photo.

### PR DESCRIPTION
**Pull request by dkinzer**
REF CIVIC-6577

 ## Description
replace the Obama image with a new, neutral image

 ## QA Steps
- [ ] Verify image has been replaced: https://ba15eb9d-e36c-48e4-821f-df693ed795d7--pr-2071.probo.build/story/end-chickenpox-varicella-vaccines-success-story

### CODE REVIEW

File: modules/dkan/dkan_fixtures/modules/dkan_default_content/data/dkan_data_story_show%3Fid=e27d899b-6d72-49c9-8807-b49424c4cf70.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
                 "id": "80a9cc2b-c817-405e-9591-14565759defc"
             }
         ],
-        "field_image": "files/dkan_data_story/A_nurse_vaccinates_Barack_Obama_against_H1N1_0.jpg",
+        "field_image": "files/dkan_data_story/immunization-of-children-in-doctors-office-725x479.jpg",
         "field_image_alt": "Nurse vaccinates Pres. Obama",
```
Comment (kducharm) [inline modules/dkan/dkan_fixtures/modules/dkan_default_content/data/dkan_data_story_show%3Fid=e27d899b-6d72-49c9-8807-b49424c4cf70.json]: Need to remove "obama" from the alt description

Comment (dkinzer) [reply modules/dkan/dkan_fixtures/modules/dkan_default_content/data/dkan_data_story_show%3Fid=e27d899b-6d72-49c9-8807-b49424c4cf70.json]: oops.

**Comment by kducharm** [comment]
@dkinzer Code review complete, couldn't merge due to tests not re-running - merge when they are completed

**Comment by dkinzer** [comment]
Test were complete and green, but status was yellow do to GitHub issues. Merged.

**Pull request merged**